tests/parm/input_global_hafs.nml.IN

```diff
@@ -231,8 +231,8 @@
        do_ca          = .false.
        ca_sgs         = .false.
        nca            = 1
-       scells         = 2600
-       tlives         = 1800
+       ncells         = 5
```

Comment:
Is there a reason for not using `@[NCELLS]` here, and in `input_nest_hafs.nml.IN` and `input_regional_hafs.nml.IN`, similar to `cpld_control.nml.IN`?

Reply:
No, I just wrote them in the same way the namelist parameters were already listed. do_ca is false in all the hafs namelists so the whole block could be removed. The variables have default values (false) in GFS_typedefs.F90

Reply:
I see. So, even when `do_ca` is false, `ncells` and other variables need to be defined. I think it's still a good idea to use `NCELLS`, and etc. defined in `default_vars.sh`, but maybe it can be done in the future since RT's already started.

Reply:
No all the CA related namelist variables have a default value in GFS_typedefs.F90. So for the HAFS namelists, the whole CA block could be removed without any change in the results. If you want I can do that change, and also add default values in default_vars.sh the next time you have a PR that doesn't change the results.

Reply:
Okay. I think they need to be removed since they serve no purpose. Let's have that change in a future PR.

Reply:
Yes, I agree. I just updated all the places that had scells etc for this PR.